src/items/functions/item_parse.cpp

```diff
@@ -0,0 +1,825 @@
+                        valueAttribute.as_string());
+		}
+	}
+}
+
+void ItemParse::parseDescription(const std::string& tmpStrValue, pugi::xml_attribute valueAttribute, ItemType& itemType) {
+	std::string stringValue = tmpStrValue;
+	if (stringValue == "description") {
+		itemType.description = valueAttribute.as_string();
+	}
+}
+
+void ItemParse::parseRuneSpellName(const std::string& tmpStrValue, pugi::xml_attribute valueAttribute, ItemType& itemType) {
+	std::string stringValue = tmpStrValue;
+	if (stringValue == "runespellname") {
+		itemType.runeSpellName = valueAttribute.as_string();
+	}
+}
+
+void ItemParse::parseWeight(const std::string& tmpStrValue, pugi::xml_attribute valueAttribute, ItemType& itemType) {
+	std::string stringValue = tmpStrValue;
+	if (stringValue == "weight") {
+		itemType.weight = pugi::cast<uint32_t>(valueAttribute.value());
+	}
+}
+
+void ItemParse::parseShowCount(const std::string& tmpStrValue, pugi::xml_attribute valueAttribute, ItemType& itemType) {
+	std::string stringValue = tmpStrValue;
+	if (stringValue == "showcount") {
+		itemType.showCount = valueAttribute.as_bool();
+	}
+}
+
+void ItemParse::parseArmor(const std::string& tmpStrValue, pugi::xml_attribute valueAttribute, ItemType& itemType) {
+	std::string stringValue = tmpStrValue;
+	if (stringValue == "armor") {
+		itemType.armor = pugi::cast<int32_t>(valueAttribute.value());
+	}
+}
+
+void ItemParse::parseDefense(const std::string& tmpStrValue, pugi::xml_attribute valueAttribute, ItemType& itemType) {
+	std::string stringValue = tmpStrValue;
+	if (stringValue == "defense") {
+		itemType.defense = pugi::cast<int32_t>(valueAttribute.value());
+	}
+}
+
+void ItemParse::parseExtraDefense(const std::string& tmpStrValue, pugi::xml_attribute valueAttribute, ItemType& itemType) {
+	std::string stringValue = tmpStrValue;
+	if (stringValue == "extradef") {
+		itemType.extraDefense = pugi::cast<int32_t>(valueAttribute.value());
+	}
+}
+
+void ItemParse::parseAttack(const std::string& tmpStrValue, pugi::xml_attribute valueAttribute, ItemType& itemType) {
+	std::string stringValue = tmpStrValue;
+	if (stringValue == "attack") {
+		itemType.attack = pugi::cast<int32_t>(valueAttribute.value());
+	}
+}
+
+void ItemParse::parseRotateTo(const std::string& tmpStrValue, pugi::xml_attribute valueAttribute, ItemType& itemType) {
+	std::string stringValue = tmpStrValue;
+	if (stringValue == "rotateto") {
+		itemType.rotateTo = pugi::cast<int32_t>(valueAttribute.value());
+	}
+}
+
+void ItemParse::parseWrapContainer(const std::string& tmpStrValue, pugi::xml_attribute valueAttribute, ItemType& itemType) {
+	std::string stringValue = tmpStrValue;
+	if (stringValue == "wrapcontainer") {
+		itemType.wrapContainer = valueAttribute.as_bool();
+	}
+}
+
+void ItemParse::parseImbuingSlot(const std::string& tmpStrValue, pugi::xml_attribute valueAttribute, ItemType& itemType) {
+	std::string stringValue = tmpStrValue;
+	if (stringValue == "imbuingslot") {
```

Comment:
This breaks compatibility with previous items.xml, introduces extra changes in the XML, and feels less correct. Please reevaluate reverting attribute name to `imbuingslots`

Reply:
This is not necessary, the imbuement will be completely redone in a pr that I will open soon, all that will change.

Reply:
![image](https://user-images.githubusercontent.com/8551443/140242201-c6ab2284-3728-4d5d-b24a-81bcee582037.png)

Reply:
When moved towards certain area (depot on my map) my client crashed with:
```
Error while processing network packet field has more than one zero id appearance
```
And a chunk of data reported:
```
6DF40167020703F50167020766980100FF98010305712400FF660300FF660300FF660300FF660300FF6603B911360100FF6603BD113A0100FF6A00BB11260E00FF6A00B61100FF6A00B611260E00FF6A00B71100FF6A25A51200FF672500FF980100FFB8016B0804FFF70400FF980126092E8500FF9801858C00FF980161000000000038000080020400482E4C2E640383001F721372000000FF006400000000000201FF000000FF9801160900FF980100FF980100FF980105FF810400FF800400FF800400FF800400FF800400FFBC1349FF
```

Not excluding that I have messed something up during merge, though, but maybe I have inadvertedly triggered some edge case. Trying to log in again triggers the same crash.